test/e2e/secrets_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"testing"
 	"time"
 
+	config "github.com/openshift/api/config/v1"
```

Comment:
s/config/oconfig

Reply:
Why? This should be done only if we have a conflict with using `config`. Places where we use `oconfig` is because we have a conflict with `config`. It is not a hard and fast rule.